daml-lf/scenario-interpreter/src/main/scala/com/digitalasset/daml/lf/Error.scala

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+package scenario
+
+import com.daml.lf.data.Ref.{Identifier, Party}
+import com.daml.lf.data.Time
+import com.daml.lf.ledger.EventId
+import com.daml.lf.speedy.SError.SError
+import com.daml.lf.transaction.{GlobalKey, Transaction}
+import com.daml.lf.value.Value.ContractId
+
+import scala.util.control.NoStackTrace
+
+/** Errors from scenario interpretation. */
+sealed abstract class Error
+    extends RuntimeException
+    with NoStackTrace
+    with Product
+    with Serializable
+
+object Error {
+
+  final case class RunnerException(err: SError) extends Error
+
+  final case class Internal(reason: String) extends Error {
+    override def toString = "CRASH: " + reason
```

Comment:
Is this necessary? Looks like `Pretty` already covers this.

Reply:
Pretty does not handle Internal Error.

Reply:
Ooops, sorry, I saw [this](https://github.com/digital-asset/daml/pull/10075/files#diff-da8d271bbcb2258273f76c8741a61b50e78d85bc4cb49f371c1d44cbf55fa354R32-R33) and thought it did, my bad.